spec/mongoid/contextual/geo_near_spec.rb

```diff
@@ -158,6 +158,27 @@
       end
     end
 
+    context "when providing a distance field" do
+
+      let(:criteria) do
+        Bar.all
+      end
+
+      let(:geo_near) do
+        described_class.new(collection, criteria, [ 52, 13 ])
+      end
+
+      let(:results) do
+        geo_near.distance_field("distance").entries
+      end
+
+      it "returns calculated distances" do
+        expect(results.first.distance.calculated).to eq(0.390512483795333)
+        expect(results.last.distance.calculated).to eq(0.460977222864644)
```

Comment:
I don't feel comfortable adding this kind of check as it's one of those "tests that just start failing at some point" thing.

Shouldn't we instead check that a distance is provided, that it is a float, and that the values are not equal as that seems to be the only thing we can be sure of from the setup?

Reply:
If we don't want to add in the exact distances here, the approach you describe sounds like a good alternative. Equally, there could also be a check to ensure that the first result's distance < last result's distance in addition/to replace a check that the values aren't equal.

Reply:
That sounds like a good option, what do you think @comandeo?

Reply:
```
        expect(results.first.distance.calculated).to be_within(0.0001).of(0.39051)
        expect(results.last.distance.calculated).to be_within(0.0001).of(0.46097)
```